tests/python/gpu/test_operator_gpu.py

```diff
@@ -2130,6 +2130,68 @@ def test_bilinear_sampler_versions():
                 if req_dict['grid'] is 'write':
                     assert_almost_equal(exe.grad_dict['grid'].asnumpy(), exe_list[ref_idx].grad_dict['grid'].asnumpy(), rtol=1e-3, atol=1e-5)
 
+@with_seed()
```

Comment:
The tests defined in `test_operator.py` will be run with `ctx=mx.gpu()` in CI. There is no need to duplicate test code in this file.

Reply:
Sorry, I've removed these redundant tests in 3cdfcf582cb522b88938128c5c8025b1562267e5.